jest.config.js

```diff
@@ -1,10 +1,49 @@
 module.exports = {
-  preset: "ts-jest",
-  testRunner: "jest-circus/runner",
```

Comment:
This isn't copied to the global config, but I think it might be the default now? @zioroboco do you know?